modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -123,9 +123,11 @@
      "SERIES_NOT_UPDATED":  "The series could not be saved",
      "SERIES_NOT_UPDATED_ID":  "The series {{ id }} could not be saved",
      "SERIES_NOT_UPDATED_ALL":  "None of the series could be saved",
-     "EVENTS_DELETED":      "The event has been deleted",
-     "EVENTS_NOT_DELETED":  "The event(s) could not be deleted",
-     "EVENTS_NOT_DELETED_NOT_AUTHORIZED":  "The event could not be deleted, because you don't have the permission to do so.",
+     "EVENT_DELETED":      "The event has been deleted.",
+     "EVENT_WILL_BE_DELETED": "The event will be unpublished first, and will then be deleted automatically.",
```

Comment:
@krinnewitz Just say "The events are being deleted.". Finally, this is what is happening.